packages/react-lowcode/src/codegen/index.ts

```diff
@@ -75,6 +80,10 @@ export function isSelectedDataTable(sourceCode:string, position: SourceLineCol){
     return isDataTableWidget(sourceCode, position)
 }
 
+export function isSelectedFormWidget(sourceCode:string, position: SourceLineCol){
```

Comment:
What is the purpose of this facadade function? I would be interesed in example sequience of necessary calls.

Reply:
The same as `isSelectedDataTable`: I need to check in demo inside clone event handler that im currently inside generated table/form and i need to add column/widget instead of simple clone selected element.